Keyboards/KeyboardsBase/ColorVariables.swift

```diff
@@ -7,128 +7,24 @@
-
-  static let commandKeyColorLight = UIColor.scribeBlueLight
-  static let commandBarColorLight = UIColor(red: 255.0/255.0, green: 255.0/255.0, blue: 255.0/255.0, alpha: 1.0)
-  static let commandBarBorderColorLight = UIColor(
-    red: 203.0/255.0, green: 203.0/255.0, blue: 206.0/255.0, alpha: 1.0
-  ).cgColor
-
-  static let keyboardBgColorLight = UIColor(red: 206.0/255.0, green: 210.0/255.0, blue: 217.0/255.0, alpha: 1.0)
-  static let keyShadowColorLight = UIColor(red: 0.0/255.0, green: 0.0/255.0, blue: 0.0/255.0, alpha: 0.35).cgColor
-
-  static let annotateRedLight = UIColor(red: 160.0/255.0, green: 24.0/255.0, blue: 35.0/255.0, alpha: 0.9)
-  static let annotateBlueLight = UIColor(red: 51.0/255.0, green: 92.0/255.0, blue: 153.0/255.0, alpha: 0.9)
-  static let annotatePurpleLight = UIColor(red: 113.0/255.0, green: 5.0/255.0, blue: 138.0/255.0, alpha: 0.9)
-  static let annotateGreenLight = UIColor(red: 62.0/255.0, green: 122.0/255.0, blue: 71.0/255.0, alpha: 0.9)
-  static let annotateOrangeLight = UIColor(red: 248.0/255.0, green: 90.0/255.0, blue: 58.0/255.0, alpha: 0.9)
-
-  // Dark theme.
-  static let scribeBlueDark = UIColor(red: 76.0/255.0, green: 173.0/255.0, blue: 230.0/255.0, alpha: 0.9)
-
-  static let keyColorDark = UIColor(red: 67.0/255.0, green: 67.0/255.0, blue: 67.0/255.0, alpha: 1.0)
-  static let keyCharColorDark = UIColor(red: 255.0/255.0, green: 255.0/255.0, blue: 255.0/255.0, alpha: 0.9)
-  static let specialKeyColorDark = UIColor(red: 32.0/255.0, green: 32.0/255.0, blue: 32.0/255.0, alpha: 1.0)
-  static let keyPressedColorDark = UIColor(red: 46.0/255.0, green: 46.0/255.0, blue: 46.0/255.0, alpha: 1.0)
-
-  static let commandKeyColorDark = UIColor.scribeBlueDark
-  static let commandBarColorDark = UIColor(red: 0.0/255.0, green: 0.0/255.0, blue: 0.0/255.0, alpha: 1.0)
-  static let commandBarBorderColorDark = UIColor(
-    red: 75.0/255.0, green: 75.0/255.0, blue: 75.0/255.0, alpha: 1.0
-  ).cgColor
-
-  static let keyboardBgColorDark = UIColor(red: 30.0/255.0, green: 30.0/255.0, blue: 30.0/255.0, alpha: 1.0)
-  static let keyShadowColorDark = UIColor(red: 0.0/255.0, green: 0.0/255.0, blue: 0.0/255.0, alpha: 0.95).cgColor
-
-  static let annotateRedDark = UIColor(red: 252.0/255.0, green: 95.0/255.0, blue: 108.0/255.0, alpha: 0.9)
-  static let annotateBlueDark = UIColor(red: 51.0/255.0, green: 158.0/255.0, blue: 235.0/255.0, alpha: 0.9)
-  static let annotatePurpleDark = UIColor(red: 173.0/255.0, green: 110.0/255.0, blue: 237.0/255.0, alpha: 0.9)
-  static let annotateGreenDark = UIColor(red: 133.0/255.0, green: 194.0/255.0, blue: 112.0/255.0, alpha: 0.9)
-  static let annotateOrangeDark = UIColor(red: 254.0/255.0, green: 160.0/255.0, blue: 93.0/255.0, alpha: 0.9)
-}
+var annotateRed = UIColor(.annotateRed)
+var annotateBlue = UIColor(.annotateBlue)
+var annotatePurple = UIColor(.annotatePurple)
+var annotateGreen = UIColor(.annotateGreen)
+var annotateOrange = UIColor(.annotateOrange)
```

Comment:
This is beautiful :) :) I'm already looking forward to all these changes just given the deletions!